python/lsst/pipe/supertask/cmdLineFwk.py

```diff
@@ -346,13 +342,12 @@ def _executeSuperTask(self, target):
             return task.runQuantum(quantum, butler)
         except Exception:
             raise  # No worries
-        except:
+        except:  # noqa: E722
```

Comment:
This was copied from `CmdLineTask`, I think the purpose of this code is to catch non-BaseException exceptions (like `raise None`) and convert it into standard Exception. I think this is done because `multiprocessing` fails badly if it sees non-Exception exceptions.

Reply:
That might be how Python 2 worked but if I do:
```python
try:
    print("About to raise")
    raise None
except RuntimeError:
    print("Caught RuntimeError")
except Exception:
    print("Caught Exception")
except BaseException:
    print("Caught BaseException")
```
then I get:
```
About to raise
Caught Exception
```
So I think this should be catching `BaseException` if you want to trap everything including Ctrl-C.

Reply:
And I get the same answer with Python 2.

Reply:
OK, I did some experimenting, indeed behavior for `raise None` is (almost) identical between 2 and 3. The difference is for old-style classes used as exception in py2, they are not caught by BaseException or Exception. And you are right that it does not matter for py3, so I should drop that.